packages/react-components/rollup.config.js

```diff
@@ -45,6 +51,31 @@ module.exports = [
       }),
       resolve(),
       commonjs(),
+      alias({
+        entries: [
```

Comment:
@akzhysparrow  can we write a common rollup config and extend those in dev and prod config, instead of repeating this?